CHANGES

```diff
@@ -1,5 +1,6 @@
-1.6.1 (trunk):
-* Add `Cstruct.concat` and `Cstruct.append`
+1.6.1 (2014-07-11):
+* Add `Cstruct.concat` and `Cstruct.append` (#57, @pqwy)
+* Add `js_of_ocaml` stubs (#63, #64, @djs55)
```

Comment:
seems like more of a 1.7 than a point release with a whole new js_of_ocaml package and new functions...